src/App.jsx

```diff
@@ -11,11 +11,13 @@ function App() {
   return (
     <>
       <Header />
-      <ArticleSearch setSearchTerm={setSearchTerm} />
-      <Routes>
-        <Route path="/" element={<ArticleList />} />
-        <Route path="/article/:id" element={<ArticlePage />} />
-      </Routes>
+      <div className="container">
+        <Routes>
+          <Route path="/" element={<ArticleList />} />
+          <Route path="/article/:id" element={<ArticlePage />} />
+          <Route path="*" element={<Error />} />
```

Comment:
It's really good to see you've already considered a `*` route for an invalid url on your app 👏